src/osha/oira/content/user.py

```diff
@@ -41,20 +43,23 @@ def __init__(self, context, request):
             self.context_type = u"sector"
             self.context_title = context.Title()
             self.contact_name = context.contact_name
+            self.reset_url = ''
         else:
             self.context_type = u"country"
             self.context_title = aq_parent(context).Title()
             self.contact_name = context.Title()
-        prt = api.portal.get_tool("portal_password_reset")
-        reset = prt.requestReset(user.getId())
-        self.reset_url = "%s/@@reset-password/%s" % (
-            api.portal.get().absolute_url(),
-            reset["randomstring"],
-        )
+            prt = api.portal.get_tool("portal_password_reset")
+            reset = prt.requestReset(user.getId())
+            self.reset_url = "%s/@@reset-password/%s" % (
+                api.portal.get().absolute_url(),
+                reset["randomstring"],
+            )
```

Comment:
@ale-rt 
I think the whole sending of the password reset email should be skipped, since we actually do not want to create a user any more. The user will not be able to react on this email, since the password in Plone will have no effect any more (membrane plugin will be disabled).

Or at least, if a user is still created (because patching out the user creation is complicated), then we want to ignore this user and at least not send the e-mail. So the subscriber below can be dropped, too IMHO.

Reply:
So I can just remove this subscriber, right?
https://github.com/euphorie/osha.oira/blob/c98d0186a9ccd32d74a4c6cd913dc752e48cbfe3/src/osha/oira/content/user.py#L55

Reply:
Did remove what I was confident about in 5bb35e04f4fddc1a4823c82aa57faede655a24b8